tests/scripts/run-xunit-perf.sh

```diff
@@ -333,69 +283,96 @@ if [ ! -d "$testRootDir" ]; then
 
 # Deploy xunit performance packages
 cd $CORE_ROOT
-echo "CORE_ROOT dir = $CORE_ROOT"
 
 DO_SETUP=TRUE
-
 if [ ${DO_SETUP} == "TRUE" ]; then
-cp  $testNativeBinDir/../../../../../packages/Microsoft.DotNet.xunit.performance.runner.cli/1.0.0-alpha-build0040/lib/netstandard1.3/Microsoft.DotNet.xunit.performance.runner.cli.dll .
-cp  $testNativeBinDir/../../../../../packages/Microsoft.DotNet.xunit.performance.analysis.cli/1.0.0-alpha-build0040/lib/netstandard1.3/Microsoft.DotNet.xunit.performance.analysis.cli.dll .
-cp  $testNativeBinDir/../../../../../packages/Microsoft.DotNet.xunit.performance.run.core/1.0.0-alpha-build0040/lib/dotnet/*.dll .
+    $DOTNETCLI_PATH/dotnet restore $CORECLR_REPO/tests/src/Common/PerfHarness/PerfHarness.csproj                                    || { echo "dotnet restore failed."; exit 1; }
+    $DOTNETCLI_PATH/dotnet publish $CORECLR_REPO/tests/src/Common/PerfHarness/PerfHarness.csproj -c Release -o "$coreOverlayDir"    || { echo "dotnet publish failed."; exit 1; }
 fi
 
 # Run coreclr performance tests
-echo "Test root dir is: $testRootDir"
+echo "Test root dir: $testRootDir"
 tests=($(find $testRootDir/JIT/Performance/CodeQuality -name '*.exe') $(find $testRootDir/performance/perflab/PerfLab -name '*.dll'))
 
-echo "current dir is $PWD"
-rm measurement.json
+if [ -f measurement.json ]; then
+    rm measurement.json || exit $EXIT_CODE_EXCEPTION;
+fi
+
 for testcase in ${tests[@]}; do
+    directory=$(dirname "$testcase")
+    filename=$(basename "$testcase")
+    filename="${filename%.*}"
 
-test=$(basename $testcase)
-testname=$(basename $testcase .exe)
-echo "....Running $testname"
-cp $testcase .
-cp $testcase-*.txt .
-
-chmod u+x ./corerun
-echo "./corerun Microsoft.DotNet.xunit.performance.runner.cli.dll $test -runner xunit.console.netcore.exe -runnerhost ./corerun -verbose -runid perf-$testname"
-./corerun Microsoft.DotNet.xunit.performance.runner.cli.dll $test -runner xunit.console.netcore.exe -runnerhost ./corerun -verbose -runid perf-$testname
-echo "./corerun Microsoft.DotNet.xunit.performance.analysis.cli.dll perf-$testname.xml -xml perf-$testname-summary.xml"
-./corerun Microsoft.DotNet.xunit.performance.analysis.cli.dll perf-$testname.xml -xml perf-$testname-summary.xml
-if [ "$uploadToBenchview" == "TRUE" ]
-    then
-	python3.5 ../../../../../tests/scripts/Microsoft.BenchView.JSONFormat/tools/measurement.py xunit perf-$testname.xml --better desc --drop-first-value --append
-fi
+    test=$(basename $testcase)
+    testname=$(basename $testcase .exe)
+
+    cp $testcase .                    || exit 1
+    if [ stat -t "$directory/$filename"*.txt 1>/dev/null 2>&1 ]; then
+        cp "$directory/$filename"*.txt .  || exit 1
+    fi
+
+    # TODO: Do we need this here.
+    chmod u+x ./corerun
+
+    echo "----------"
+    echo "  Running $testname"
+    echo "----------"
+    run_command $stabilityPrefix ./corerun PerfHarness.dll $test --perf:runid Perf --perf:collect stopwatch || exit 1
```

Comment:
This hardcodes `--perf:collect` and `stopwatch`, ignoring the input parameters passed to this script. Is that intentional?